hedera-node/src/test/java/com/hedera/services/utils/MerkleEntityIdUtilsTest.java

```diff
@@ -97,59 +100,22 @@ public void serializesExpectedSolidityAddress() {
 		assertEquals(equivContract, contractParsedFromSolidityAddress(actual));
 	}
 
-	@Test
-	public void translatesIndexOutOfBoundsException() {
-		// given:
-		String invalidLiteral = "0.1";
-
-		IllegalArgumentException iae = null;
-
-		// when:
-		try {
-			accountParsedFromString(invalidLiteral);
-		} catch (IllegalArgumentException thrown) {
-			iae = thrown;
-		}
-
-		// then:
-		assertNotNull(iae.getCause());
-		assertEquals(ArrayIndexOutOfBoundsException.class, iae.getCause().getClass());
-	}
-
-	@Test
-	public void translatesNumberFormatException() {
-		// given:
-		String invalidLiteral = "0.0.asdf";
-
-		IllegalArgumentException iae = null;
-
-		// when:
-		try {
-			accountParsedFromString(invalidLiteral);
-		} catch (IllegalArgumentException thrown) {
-			iae = thrown;
-		}
-
-		// then:
-		assertNotNull(iae.getCause());
-		assertEquals(NumberFormatException.class, iae.getCause().getClass());
+	@ParameterizedTest
+	@CsvSource({"0", "0.a.0", "...", "1.2.3.4", "1.2.three", "1.2.333333333333333333333333333333333333333333"})
+	void rejectsInvalidAccountLiterals(String badLiteral) {
+		// expect:
+		assertThrows(IllegalArgumentException.class, () -> parseAccount(badLiteral));
```

Comment:
Should assert also the exception error message

Reply:
Added assert for error message.